api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java

```diff
@@ -521,4 +539,15 @@ private static String convertNamedParameterToString(
         throw new BadRequestException("Language not supported");
     }
   }
+
+  private ConceptSet handlePrePackagedSurveyConceptSet() {
+    List<Long> conceptIds = conceptBigQueryService.getSurveyQuestionConceptIds();
+    ConceptSet surveyConceptSets = new ConceptSet();
+    surveyConceptSets.setName("PrePackaged Survey");
```

Comment:
This text should be something more descriptive, like, "All Surveys" or "All Survey Questions".

Reply:
Yes i forgot to change here. it should be All Surveys